onnx2torch/utils/safe_shape_inference.py

```diff
@@ -12,7 +12,11 @@ def _is_big_model(model: ModelProto) -> bool:
     return model.ByteSize() / (1024 * 1024 * 1024) > 2.0
 
 
-def _shape_inference_by_model_path(model_path: Union[Path, str], output_path: [Path, str], **kwargs) -> ModelProto:
+def _shape_inference_by_model_path(  # pylint: disable=missing-function-docstring
+    model_path: Union[PosixPath, str],
+    output_path: Union[PosixPath, str],
+    **kwargs,
+) -> ModelProto:
```

Comment:
Why you replace Path by PosixPath? What is the reason?

Reply:
@ivkalgin Sorry, I've been misrepresenting `PosixPath` and `Path`, I'll change it back here